cmd/cloud/main.go

```diff
@@ -45,3 +61,20 @@ func NewCloudCommand() *cobra.Command {
 
 	return cmd
 }
+
+func RequireAccessTokenCheck(cmd *cobra.Command, args []string) error {
```

Comment:
Don't think we need to expose this.


```suggestion
func requireAccessTokenCheck(cmd *cobra.Command, args []string) error {
```